### DISCUSSION TITLE
500 Responses from the API cause failure in app

### DISCUSSION THREAD

**Discussion by just-be-dev** [marcomontalbano/figma-export#120] [Q&A] [answered]
I didn't open this as an issue just given it seems like it's an issue on Figma's side.

I've got a fairly standard setup

```js
module.exports = {
  commands: [
    [
      'components',
      {
        fileId: 'XXXXXX',
        onlyFromPages: ['Icons'],
        transformers: [
          require('@figma-export/transform-svg-with-svgo')({
            plugins: [],
          }),
        ],
      },
    ],
  ],
}
```

When I run it I'm getting this response

```
Error: while fetching fileImages: Request failed with status code 500
    at /home/zephraph/oxide/console/node_modules/@figma-export/core/dist/lib/figma.js:94:15
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

That maps over to the [`fileImages`](https://github.com/marcomontalbano/figma-export/blob/86d21aaaa56f3f6024461210e8cd18bb82fee39c/packages/core/src/lib/figma.ts#L75) function. I tried to see what my be triggering that from our side. There are 125 IDs being passed. I tried to truncate that down to 50 and it still failed. 20 also failed. And 1. 

So, I think there's just an issue w/ their `GET /v1/images/` API. Could someone else verify?

**Comment by just-be-dev** [comment, marked as answer]
welp, it looks like either generation a new API token resolved this _or_ they coincidentally fixed whatever was failing behind the scenes. 🤷

**Reply by marcomontalbano** [reply]
Hi @zephraph, I was writing to you just now :)

Thanks for the update. I just tested with a file containing many components and it worked.
As you said probably it was a temporary issue on Figma.

If you notice this again let me know!
